platforms/android/demo/build.gradle

```diff
@@ -2,14 +2,16 @@ apply plugin: 'com.android.application'
 
 android {
   compileSdkVersion 26
-  buildToolsVersion '25.0.3'
+  buildToolsVersion '26.0.2'
 
   def apiKey = project.hasProperty('mapzenApiKey') ? mapzenApiKey : System.getenv('MAPZEN_API_KEY')
 
   defaultConfig {
     minSdkVersion 15
     targetSdkVersion 26
     buildConfigField 'String', 'MAPZEN_API_KEY', '"' + apiKey + '"'
+    // provide dimension strategy for tangram library's dim
+    missingDimensionStrategy 'tangramAbi', 'slim'
```

Comment:
I'm not totally clear on how flavors are published from a library like ours. Will all projects using `tangram` need to add this `missingDimensionStrategy`? Or just projects using the 3.0 plugin? Will this make any difference for projects using older plugin versions?